main_app/templates/main_app/women_laws.html

```diff
@@ -59,6 +59,15 @@
             margin-left: auto;
             margin-right: auto;
         }
+        .description{
+            margin-left: 0.5rem;
+            margin-right: 0.5rem;
+            border-left: 3px solid #a8dadc;
+            padding-left: 0.5rem;
```

Comment:
Please add padding-right as well for this which is equal to padding-left. Also, increase the padding-left as well.